edge/pkg/metamanager/process.go

```diff
@@ -492,6 +515,10 @@ func (m *metaManager) process(message model.Message) {
 		constants.CSIOperationTypeControllerPublishVolume,
 		constants.CSIOperationTypeControllerUnpublishVolume:
 		m.processVolume(message)
+	case constants.OperationTypeGetServiceAccount:
```

Comment:
Is it possible to reuse the `QueryOperation` here?

Reply:
`QueryOperation`  involves the operation of the `database`, and it reduces codes readability if we directly modify in the `QueryOperation` related funtions , So IMO, it's better to just add one more operation to process `serviceaccount token` exclusively

Reply:
> QueryOperation involves the operation of the database

The result should be stored in the database.

> it's better to just add one more operation to process serviceaccount token exclusively

It's better to reuse the lib func instead of adding the redundant code.

Reply:
ok

Reply:
done